src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
Don't forget that i cannot test it, no NVIDIA, so don't be upset after me if that doesn't work...
```
ffmpeg64.exe -y -i jellyfish-200-mbps-4k-uhd-hevc-10bit.mkv -bufsize 6M -maxrate 6M -b:v 3M -filter:v "hwupload_cuda,format=yuv420p,interp_algo=lanczos,hwdownload" -rc cbr_hq
 -c:v h264_nvenc -preset superfast -profile:v main -c:a aac -b:a 320k -ar 48000 -f ismv -movflags isml+frag_keyframe hevc_test1.mp4 -benchmark
```
```
ffmpeg64.exe -y -i jellyfish-200-mbps-4k-uhd-hevc-10bit.mkv -bufsize 6M -maxrate 6M -b:v 3M -filter:v "hwupload_cuda|format=yuv420p|interp_algo=lanczos|hwdownload" -rc cbr_hq
 -c:v h264_nvenc -preset superfast -profile:v main -c:a aac -b:a 320k -ar 48000 -f ismv -movflags isml+frag_keyframe hevc_test1.mp4 -benchmark
```